static/app/app/list/views.js

```diff
@@ -21,7 +21,8 @@ var LunchMoveView = Marionette.ItemView.extend({
     },
     events: {
         'click @ui.addMove': 'addMove',
-        'click @ui.editMove': 'addMove'
+        'click @ui.editMove': 'addMove',
+        'click .delete': 'deleteMove'
```

Comment:
In our practice, we like to keep DOM class names mostly for styling. So when we attach functionality to a DOM element, we usually don't reference it via a class selector but rather try to do it using html data attributes and the [Marionette UI hash](http://marionettejs.com/docs/v2.4.3/marionette.view.html#ui-interpolation).

This makes it clear that something is being used as an element for user interaction and also allows someone else to make changes to the styling while being sure they won't accidentally break the functionality.

So for this:
 - add a `data-ui="delete"` attribute to the html for the delete button.
 - add an entry to the ui hash, using the data attribute as the selector `delete: '[data-ui="delete"]'`
 - reference the ui element in the event: `'click @ui.delete': 'deleteMove'`.

Lemme know if you have questions about those